src/app/features/event/event-map/event-map.component.ts

```diff
@@ -0,0 +1,58 @@
+import { CommonModule } from '@angular/common';
+import { ChangeDetectionStrategy, Component, inject, Input } from '@angular/core';
+import * as leaflet from 'leaflet';
+import { EventMapMarkerService } from './event-map-marker.service';
+
+@Component({
+  selector: 'app-event-map',
+  standalone: true,
+  imports: [CommonModule],
+  templateUrl: `./event-map.component.html`,
+  styleUrls: ['./event-map.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class EventMapComponent {
+  private map: leaflet.Map | leaflet.LayerGroup<any> | undefined;
+  private eventMapMarkerService = inject(EventMapMarkerService);
+  isCoordinates = true;
+  @Input() coordinates!: { altitude: number | null; latitude: number | null };
```

Comment:
Jak mówisz mu, że coordinates będzie na pewno, to można to też dopisać w selectorze: `app-event-map[coordinates]` wtedy wywali błąd, jeżeli nie dodamy tego propsa